lom/src/plugins/vendors/arista/arista_common/gnmi_server.go

```diff
@@ -0,0 +1,839 @@
+/* gnmi_server.go
+ *
```

Comment:
Why can't it be named as ..._test.go, to make it explicit.
And why is main commented out.

My concern: It does not become part of Arista agent bloating binary size for no value.

Reply:
This is consumed in unit tests. Main is comented because, it can also run as standalone.

Reply:
This will not go in binary as its only imported in UT's